Frontend/CO.CDP.OrganisationApp/Pages/OneLogin.cshtml.cs

```diff
@@ -66,10 +68,21 @@ private async Task<IActionResult> UserInfo()
                 session.Set(Session.UserDetailsKey, ud);
             }
 
+            var personInviteId = session.Get<Guid?>("PersonInviteId");
+
+            if (personInviteId != null)
+            {
+                await ClaimPersonInvite(person.Id, personInviteId.Value);
```

Comment:
Person should not be allowed to do any operation without accepting PrivacyPolicy

Reply:
I need a bit more insight on this one. When do they usually accept the privacy policy? Is it just before adding their first organisation during the normal flow?

Reply:
When person login first time, they have to go through PrivacyPolicy